tests/ErrorsContainer.test.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { ErrorsContainer } from '../src/components';
+import { ErrorField } from '../src/components/separate';
+
+describe('ErrorsContainer', () => {
```

Comment:
I think you should add test with case when you will receive errors from ObjectField and ListField.
#### Example
```js
{
    foo: [
        { 
            bar: ['some error'],
            foo: [
                bar: ['some error2']
            ]
        }
    ]
}
```

Reply:
Done